MAINTAINERS.yml

```diff
@@ -1220,7 +1220,7 @@ MCU Manager:
         - "area: mcumgr"
 
 OSDP:
-    status: odd fixes
+    status: maintained
     collaborators:
         - sidcha
```

Comment:
> subsys/mgmt/osdp _has_ a dedicated maintainer who will rush to address
any issues identified :).

In that case, please add yourself as a maintainer.

Reply:
Done.